### PR TITLE
Bug 1878289: connectivity check events have ambiguous related object

### PR DESCRIPTION
**Pull request by sanchezl**

### CODE REVIEW

File: bindata/v4.1.0/kube-apiserver/check-endpoints-clusterrole.yaml [hunk 1]
```diff
@@ -30,3 +30,9 @@ rules:
       - get
       - list
       - watch
+  - resources:
+      - events
+    apiGroups:
+      - ""
+    verbs:
+      - create
```
Comment (deads2k) [inline bindata/v4.1.0/kube-apiserver/check-endpoints-clusterrole.yaml:38]: get, list, watch, create, update, patch

File: bindata/v4.1.0/kube-apiserver/check-endpoints-clusterrolebinding-node-reader.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: system:openshift:controller:kube-apiserver-check-endpoints-node-reader
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: system:node-reader
```
Comment (deads2k) [inline bindata/v4.1.0/kube-apiserver/check-endpoints-clusterrolebinding-node-reader.yaml:8]: link to where this is defined? I don't recognize it.

**Comment by deads2k** [comment]
/lgtm
/hold

looks fine.  needs to be green though and fix the role.  release hold and reapply when ready.

File: bindata/v4.1.0/kube-apiserver/check-endpoints-clusterrolebinding-node-reader.yaml (hunk 2 above)
Comment (sanchezl) [reply bindata/v4.1.0/kube-apiserver/check-endpoints-clusterrolebinding-node-reader.yaml:8]:
```
Name:         system:node-reader
Labels:       kubernetes.io/bootstrapping=rbac-defaults
Annotations:  rbac.authorization.kubernetes.io/autoupdate: true
PolicyRule:
  Resources      Non-Resource URLs  Resource Names  Verbs
  ---------      -----------------  --------------  -----
  nodes/stats    []                 []              [create get]
  nodes          []                 []              [get list watch]
  nodes/metrics  []                 []              [get]
  nodes/spec     []                 []              [get]
```

I found this: https://github.com/openshift/openshift-apiserver/blob/ed260ac7ae860d493187963ce23efabb36a2884f/pkg/cmd/server/bootstrappolicy/policy.go#L411-L434

Comment (deads2k) [reply bindata/v4.1.0/kube-apiserver/check-endpoints-clusterrolebinding-node-reader.yaml:8]: > I found this: https://github.com/openshift/openshift-apiserver/blob/ed260ac7ae860d493187963ce23efabb36a2884f/pkg/cmd/server/bootstrappolicy/policy.go#L411-L434

this repo cannot depend on openshift-apiserver-operator like that.

Comment (deads2k) [reply bindata/v4.1.0/kube-apiserver/check-endpoints-clusterrolebinding-node-reader.yaml:8]: create a second clusterrole if need be

**Pull request merged**